bundle/config/mutator/resourcemutator/apply_target_mode_test.go

```diff
@@ -27,6 +25,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// FIXME apply_target_mode_test should be untied from apply_presets_test
```

Comment:
What does it mean to untie? Test mutators separately? In general, this diminishes the value of the test if there are mutators that tightly coupled. Acceptance test that checks the whole validate phase is a better approach there.

Reply:
There is a lot of coupling between tests, they use private methods from each other, e.g. to generate test data

Reply:
I think this makes them good candidates for acceptance tests which do not care about implementation and thus enable refactoring. Making unit tests for each of these just ties us more to this implementation.